lochness/redcap/__init__.py

```diff
@@ -333,12 +333,23 @@ def sync(Lochness, subject, dry=False):
             logger.debug("Downloading REDCap data")
             _debug_tup = (redcap_instance, redcap_project, redcap_subject)
 
-            record_query = {
-                'token': api_key,
-                'content': 'record',
-                'format': 'json',
-                'records': redcap_subject
-            }
+            if 'UPENN' in redcap_instance:
+                # UPENN REDCap is set up with its own record_id, but have added
+                # "session_subid" field to note AMP-SCZ ID
+                record_query = {
+                    'token': api_key,
+                    'content': 'record',
+                    'format': 'json',
+                    'filterLogic': f"[session_subid] = '{redcap_subject}'"
```

Comment:
Shouldn't the last element be `subject.id`?

Reply:
Second question:

> r = requests.post(api_url, data=data, stream=True, verify=False)

Does a REDCap query recognize `filterLogic`?

Reply:
Here `redcap_subject` is also the ID. REDCap query recognizes `filterLogic`. I've check this PR in Pronet server with YA00015 and it's fully working.